docs/modeling-guide.md

```diff
@@ -469,3 +469,35 @@ export interface Request<TDocument> extends RequestBase {
   body?: TDocument
 }
 ```
+
+#### `@security_prerequisites_index`
+
+If an endpoint has some index security prerequisites to satisfy, you can specify them here.
+
+```ts
+/**
+ * @rest_spec_name indices.create
+ * @since 0.0.0
+ * @stability stable
+ * @security_prerequisites_index create_index manage
```

Comment:
Please use as list separator the commata `,`. That overlaps how `@aliases` is used.

Reply:
Good catch!